### PR TITLE
crude first edition

### PR DESCRIPTION
**Pull request by PrincetonPoh**
What do you think of this crude template and the approach?

Once the template is g, i'll do up some examples for further refining

### CODE REVIEW

File: _notes/rft.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: Request For Teachers
+---
+
+I believe that the RFT should be approached from the "problem" angle. 
+Meaning the teachers should be reaching for a RFT whenever they see a problem.
+Here are some examples off the top of my head:
+
+- Some of my students have a hard time coming up with compelling copywriting for their product launch
```
Comment (ks0m1c) [inline _notes/rft.md]: I think its better to start with 
Students currently [present context], but [the gap in context]. Ideally they should be able to [the ideal if the gap is crossed]

e.g Students currently have 3d printers in the homeroom but they lack the practice around CAD tooling. Ideally they are able to print simple everyday items after.

File: _notes/rft.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+Write a catchy title here
+e.g 
+
+## 2. Problem
+Describe what is the problem you're facing
+e.g 
+
+## 3. Context
+Give some context as to why you're facing this problem
+e.g
```
Comment (ks0m1c) [inline _notes/rft.md]:
```
Current Context: Students currently understand fundamental of graphics and have knowledge of lighting scenes and 3d modelling.

Identified Gap: However, they lack practical insight into procedural node based programming as being applied in real-world world of animation and 3D visualisation

Ideal Outcome: Ideally, students should be able to construct a simple 3d scene with programmable elements, that sets them up in good stead for their [3d digital game art project](link to past works)
```

Comment (ks0m1c) [reply _notes/rft.md]: ## **Current Context**: Students currently understand fundamental of graphics and have knowledge of lighting scenes and 3d modelling.

## **Identified Gap**: However, they lack practical insight into procedural node based programming as being applied in real-world world of animation and 3D visualisation

## **Ideal Outcome**: Ideally, students should be able to construct a simple 3d scene with programmable elements, that sets them up in good stead for their [3d digital game art project](link to past works)

**Review by ks0m1c — COMMENTED** [review]
i dont think the RFT should be too many steps at most 3-5

File: _notes/rft.md [hunk 3]
```diff
@@ -2,68 +2,44 @@
...
 
-## 4. What have you tried?
-This is to minimize falling into the same trap
-e.g I gave them some Youtube tutorial to guide them through but it's not effective.
-- 
+Students currently [Current Context]. 
+However, [Identified Gap]. 
+Ideally they should be able to [Ideal Outcome].
+
+Optional details:
```
Comment (PrincetonPoh) [inline _notes/rft.md]: I think we can squeeze everything else into this part that is optional.

**Comment by PrincetonPoh** [comment]
This should be quite distilled already. Lmk if got any other improvement thoughts @ks0m1c

File: _notes/rft.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+------------------------------------------------------------------------------------
+# Mentor Profile Template
+
+
+{% include figure image_path="assets/lib/square_ikigai.png" alt="" %}
+
+1. Fullname?
+2. If asked to MENTOR students, which areas would you be willing to cover? (indicate your skills & interests)
+3. Beyond just mentoring, if you're asked to TEACH a class of students about anything you wish, what would it be?
```
Comment (ks0m1c) [inline _notes/rft.md:80]: What is something you had always wanted to teach

 has something to it

or something you had always wanted others to learn about

File: _notes/rft.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+------------------------------------------------------------------------------------
+# Request for Learners (RFL) Template
+
+I want to help students [long-term outcome] through [creative method].
+
+Optional Details:
+- This is in response to which RFT?
+- Please list your key lesson objectives
+- What is your exact lesson plan?
```
Comment (ks0m1c) [inline _notes/rft.md]: build your lesson plan

instead of interrogating we are helping them scaffold

Comment (ks0m1c) [reply _notes/rft.md]: nested lets design an activity or two

File: _notes/rft.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+- What is your exact activity plan?
+- What are some measurable metrics you would like to track?
```
Comment (ks0m1c) [inline _notes/rft.md]: i feel for measure it has to tie in with feedback 

how do you wish for students to respond with various time bounds
in a week 
in a month 
by the end of the term etc

**Review by ks0m1c — CHANGES_REQUESTED** [review]
looking good especially the sst ones

**Pull request merged**